target/snitch_cluster/README.md

```diff
@@ -1,24 +1,60 @@
-# Snitch cluster target
+# SNAX Cluster Target
 
+## Details
 The Snitch cluster target (`target/snitch_cluster`) is a simple RTL testbench
-around a Snitch cluster. The cluster can be configured using a config file. By default, the config file which will be picked up is `target/snitch_cluster/cfg/default.hsjon`.
+around a SNAX cluster. It is still the same as the Snitch cluster but with SNAX modifications. The cluster can be configured using a config file. By default, the config file which will be picked up is `target/snitch_cluster/cfg/default.hsjon`.
 
 The configuration parameters are documented using JSON schema. Documentation for the schema and available configuration options can be found in `docs/schema-doc/snitch_cluster/`).
 
 The cluster testbench simulates an infinite memory. The RISC-V ELF file to be simulated is
 preloaded using RISC-V's Front-End Server (`fesvr`).
 
-## Tutorial
+## Quick Start
+
+These steps demonstrate how you quickly run a simple accelerator in SNAX shell. This assumes you either have the necessary dependencies already or have the docker started (see [getting started](ug/getting_started.md)).
+
+1. Move into `./target/snitch_cluster/.`
+
+```shell
+cd ./target/snitch_cluster/.
+```
+
+2. Build custom hardware using one of the `cfg.hjson` files for a SNAX accelerator. The code below builds the SNAX with HWPE MAC engine. Add the `-j` option to set the number of  cores to run the build. The output is a RTL Verilator binary which runs the system. 
+
+```shell
+make CFG_OVERRIDE=cfg/snax-mac.hjson bin/snitch_cluster.vlt -j 16
+```
+
+3. Build the software. This produces `.elf` files (binary) which we feed into the built RTL builds.
+
+```shell
+make DEBUG=ON sw
+```
+
+4. Feed the `.elf` file into the RTL binary. Wait for the simulation to run. This produces `.dasm` (disassembly) files stored in the `./logs` directory.
+
+```shell
+bin/snitch_cluster.vlt sw/apps/snax-mac/build/snax-mac.elf
+```
+
+5. Make traces. See (TODO add section) section. This produces a `.txt` dump which are the traces you can inspect.
```

Comment:
Can you link/make the issue for this?

Reply:
Oops I just forgot to write this but will add it later.

Reply:
Fixed this rather than making an issue.